app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarView.kt

```diff
@@ -54,21 +55,20 @@ class BrowserToolbarView(
 
     private val settings = container.context.settings()
 
+    lateinit var toolbarIntegration: ToolbarIntegration
+
+    lateinit var view: BrowserToolbar
+
+    lateinit var layout: View
```

Comment:
nit: I think this can be private (or if my suggested change below is incorrect, is unused)